css/style.css

```diff
@@ -0,0 +1,345 @@
+
+article, aside, details, figcaption, figure, footer, header, hgroup, menu, nav, section {
+	display: block;
+}
+
+body {
+	line-height: 1;
+}
+
+ol, ul {
+	list-style: none;
+}
+
+blockquote, q {
+	quotes: none;
+}
+
+blockquote:before, blockquote:after, q:before, q:after {
+	content: '';
+	content: none;
+}
+
+table {
+	border-collapse: collapse;
+	border-spacing: 0;
+}
+
+/* 
+*
+*
+*
+* base
+*
+*
+*
+*/
+
+body {
+	background-color: #f9f9f9;
+}
+
+header {
+	padding: 11px 0 11px 0;
+	background-color: #ffffff;
+}
+
+main {
+	margin: 50px;
+}
+
+h1{
+	font-weight: 800;
+	font-size: 20px;
+	font-family: "Times New Roman", Times, serif;
+}
+
+iframe {
+	width: 100%;
+	height: 450px;
+	border: none;
+}
+
+aside {
+	padding-top: 3px;
+}
+
+footer {
+	background-color: #141B38;
+	color: #ffffff;
+	text-align: center;
+	padding: 30px 0;
+}
+
+/* 
+*
+*
+*
+* layout
+*
+*
+*
+*/
+
+#grid-container-header {
+	display: grid;
+	grid-template-columns: 1fr 3fr 1fr;
+}
+
+.logo-container {
+	margin: auto;
+}
+
+.search-container {
+	margin: 0 auto;
+}
+
+.icon-container {
+	margin: auto 0;
+}
+
+.right {
+	float: right;
+	padding-right: 25px;
+}
+
+#grid-container-main {
+	display: grid;
+	grid-template-columns: 2.2fr 1fr;
+	margin-top: 24px;
+}
+
+#section-container {
+	margin-right: 23px;
+}
+
+#section-container-main h1 {
+	padding-top: 10px;
+}
+
+#grid-container-video-main-info {
+	display: grid;
+	grid-template-columns: 1fr 2fr;
+	padding-bottom: 10px;
+	padding-top: 10px;
+}
+
+#grid-container-video-main-info-extra {
+	text-align: right;
+}
+
+#grid-container-video-main-info-extra .fa {
+	padding-left: 20px;
+}
+
+#grid-container-video-additional-info {
+	display: grid;
+	grid-template-columns: 0.5fr 3fr 1fr;
+	padding-top: 10px;
+}
+
+#video-information h2{
+	padding-bottom: 3px;
+}
+
+#subscribe-button button {
+	float: right;
+}
+
+#video-description {
+	padding-top: 35px;
+}
+
+#video-description-more {
+	padding-top: 15px;
+}
+
+.grid-aside {
+	display: grid;
+	grid-template-columns: 1fr 1.2fr;
+	margin-bottom: 10px;
+}
+
+#thumbnail {
+	text-align: right;
+}
+
+.thumbnail-description {
+	padding: 3px 0 0 7px;
+}
+
+.thumbnail-info {
+	padding-top: 10px;
+	padding-bottom: 3px;
+}
+
+#first-thumbnail {
+	padding-bottom: 5px;
+}
+
+#second-thumbnail {
+	padding-top: 5px;
+}
+
+/* 
+*
+*
+*
+* template
+*
+*
+*
+*/
+
+.logo-container {
+	color: red;
+}
+
+.black {
+	color: black;
+	margin-left: 5px;
+}
+
+.fa-youtube-play {
+	margin-left: 20px;
+}
+
+.search-container input {
+	float: left;
+	border: 1px solid #D6D6D6;
+	padding: 8px;
+	width: 500px;
+	border-top-left-radius: 2px;
+	border-bottom-left-radius: 2px;
+}
+
+.search-container input::placeholder {
+	color: rgb(141, 139, 139);
+}
+
+.search-container button {
+	float: left;
+	border: 1px solid #333;
+	border-left: none;
+	padding: 8px;
+	width: 65px;
+	border-top-right-radius: 2px;
+	border-bottom-right-radius: 2px;
+	background-color: #dfdede;
+	opacity: 0.2;
```

Comment:
- [ ] Here there are some properties that are repeating as well. You can try this instead (In this case it does not seems much improvement because only a small number of properties are common, but as your projects gets bigger you might need to do like this because a lot of properties might repeat):

.search-container input {
  border: 1px solid #d6d6d6;
  padding: 8px;
  width: 500px;
  border-bottom-left-radius: 2px;
}

.search-container button,
.search-container input {
  float: left;
  padding: 8px;
}

.search-container button {
  border: 1px solid #333;
  border-left: none;
  width: 65px;
  border-bottom-right-radius: 2px;
  background-color: #dfdede;
  opacity: 0.2;
}